lms/models/lti_registration.py

```diff
@@ -37,3 +39,62 @@ class LTIRegistration(CreatedUpdatedMixin, BASE):
     application_instances = sa.orm.relationship(
         "ApplicationInstance", back_populates="lti_registration"
     )
+
+    @staticmethod
+    def urls(issuer, client_id):
```

Comment:
Does this belong to each product?

Not sure, there's not request.product here in the admin pages.

Reply:
There wouldn't be a product in play as you say, but it could be something you get from each one as an array. I think that might be overly obscure though. I've made some suggestions below to how we could get this out of the code though.